default.nix

```diff
@@ -0,0 +1,48 @@
+{ pkgs ? import <nixpkgs> {} }:
+
+with pkgs;
+
+let
+  version = "0.9.3";
+
+  nodegit = import ./nodegit.nix { inherit pkgs; };
+
+  libs = [ stdenv.cc.cc glibc ];
+
+in
+
+  stdenv.mkDerivation {
+    name = "clever-cli-${version}";
+
+    buildInputs = [ nodegit nodejs-6_x ];
```

Comment:
I'm not sure about that nodejs dep.

Reply:
*He surely made it on purpose*

![yo](https://media1.tenor.com/images/fb3f2d1e814190100a4ae401b1660d5b/tenor.gif?itemid=6081931)

Reply:
I have no clue. It made sense at the time, but I'm pretty sure we could remove it, and it should work with other node variant :thinking: